OpenOversight/app/commands.py

```diff
@@ -565,7 +566,11 @@ def bulk_add_officers(filename, no_create, update_by_name, update_static_fields)
 
 
 @click.command()
-@click.argument("department-name")
+@click.argument("department-name", required=True)
+@click.argument(
+    "department-state",
+    type=click.Choice([state.abbr for state in us.STATES]),
+)
```

Comment:
In the next PR, once we can create and edit Departments from the front-end, we can make this required.